roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -79,7 +79,11 @@ reserved_host_memory_mb = {{ nova.compute_reserve_ram }}
 reserved_host_disk_mb = {{ nova.reserved_host_disk_mb }}
 
 # Misc #
+{% if ironic.enabled -%}
+compute_driver = ironic.IronicDriver
+{% else %}
```

Comment:
I think this would preclude having an environment of both KVM and bare metal. Wouldn't it be better to check if the host that this template is being deployed on is a member of the `compute_ironic` group? Or rely on the host vars of that particular host to set `nova.compute_driver` correctly?